res/css/views/elements/_Field.scss

```diff
@@ -16,6 +16,8 @@ limitations under the License.
 
 /* TODO: Consider unifying with general input styles in _light.scss */
 
+@import "../../_font-sizes.scss";
+
```

Comment:
this is not needed, it is implicitly imported by the base style

Reply:
Nicely spotted

Reply:
done